### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][JAWS/NVDA][MAS4.2.1][IntelliJ][Wipro]: NVDA and Jaws are not announcing the content present in "Run/Debug Configurations" dialog sequentially

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1518]
**Repro Steps:**
Open and Start IntelliJ IDEA.

1. Create a new project.
2. Navigate to and select "edit configurations" present at right side of the dialog.
3. "Run/Debug Configurations" dialog will open.
4. .In "Run/Debug Configurations" Press "Insert+B" to read  the entire dialog then observe the NVDA and JAWS Behavior

**Expected Result:**
NVDA and Jaws should announce in sequential manner

- Run - Azure Web App: sample:sample  dialog  Name
- Add New Configuration (Alt+Insert)  button  Add New Configuration (Alt+Insert)
Remove Configuration (Alt+Delete)  button  Remove Configuration (Alt+Delete)
 Copy Configuration (Ctrl+D)  button  Copy Configuration (Ctrl+D)
Edit Defaults  button  Edit Defaults
 Move Up (Alt+Up)  button  Move Up (Alt+Up)
Move Down (Alt+Down)  button  Move Down (Alt+Down)
Move into new folder  button  Move into new folder
- tree view
 level 1  Azure Web App  expanded  1 of 2
 level 2  Azure Web App: sample:sample  selected  1 of 1
level 1  Defaults  collapsed  2 of 2
- Name
- Use Existing Web App  radio button  not checked  1 of 2
Create New Web App  radio button  checked  2 of 2
- Name edit field
- Subscription combo box
- Resource Group expand/collapse
  All interactive under Resorce Group
- App Service Plan  expand/collapse
 All interactive elements under App Service Plan
- Java Expand/Collapse
All interactive elements under Java
- Before launch: Launch Web Browser, Build Artifacts, Activate tool window
Add New Configuration (Alt+Insert)  button  Add New Configuration (Alt+Insert)
Remove Configuration (Alt+Delete)  button  Remove Configuration (Alt+Delete)
 Copy Configuration (Ctrl+D)  button  Copy Configuration (Ctrl+D)
Edit Defaults  button  Edit Defaults
 Move Up (Alt+Up)  button  Move Up (Alt+Up)
Move Down (Alt+Down)  button  Move Down (Alt+Down)
- List
 List items
- Show this page  check box  not checked
- Activate tool window  check box  checked
- Invalid Data: Please specify a valid docker file.
- button  Show help contents
- Run  button
- Cancel  button
- Apply  button  alt+A

**Actual Result:**
**NVDA Behavior**: NVDA is annoucing as 
Run - Azure Web App: sample:sample  dialog  Name
.azurewebsites.net
Subscription
Resource Group
App Service Plan
Location
Pricing Tier
N/A
N/A
Location
Pricing Tier
Java
Java Version
Artifact
Before launch: Activate tool window
tree view
Choose Web App  panel
table
adobe-rg  row 1  Resource group  column 4
level 1  Azure Web App  expanded  1 of 2
level 2  Azure Web App: sample:sample  selected  1 of 1
level 1  Defaults  collapsed  2 of 2
Add New Configuration (Alt+Insert)  button  Add New Configuration (Alt+Insert)
Remove Configuration (Alt+Delete)  button  Remove Configuration (Alt+Delete)
Copy Configuration (Ctrl+D)  button  Copy Configuration (Ctrl+D)
Edit Defaults  button  Edit Defaults
Move Up (Alt+Up)  button  Move Up (Alt+Up)
Move Down (Alt+Down)  button  Move Down (Alt+Down)
Move into new folder  button  Move into new folder
Sort Configurations  button  Sort Configurations
Use Existing Web App  radio button  not checked  1 of 2
Create New Web App  radio button  checked  2 of 2
Name
Run - Azure Web App: sample:sample
Azure Web App: sample:sample
**Jaws Behavior:**
Run/Debug Configurations dialog
Name JDK Web container Resource group Artifact Before launch: Launch Web Browser, Build Artifacts, Activate tool window Invalid Data: Choose a web app to deploy. 
Name: Edit
Azure Web App: untitled4:untitled4

**Platform:**
IntelliJ IDEA 2017.3.2 (Community edition)
Build# IC-173.4127.27
JRE:1.8.0_152-release-1024-b8amd64
JVM: OpenJDK 64 Bit Server VM by Jet brains.
OSBuild Version: Version 1709(Os Build 16299.192)

![hierarcy is incorrect for edit configurations window](https://user-images.githubusercontent.com/22956130/36151109-17365198-10ec-11e8-9a45-13ab7ac12a13.jpg)
![edit configu-insert-b](https://user-images.githubusercontent.com/22956130/36151116-2283cc06-10ec-11e8-8616-78d8d5d303d1.jpg)

**Comment by Eskibear** [comment]
Run/Debug Configurations page is out of the toolkit's control.

**Comment by Eskibear** [comment]
Cannot reproduce it.

According to the practice, NVDA cannot read content of any dialog in IntelliJ IDEA 2017.3.4 Community.
A ticket is created to track the issue.

Ref:
https://youtrack.jetbrains.com/issue/IDEA-191325

**Comment by sallyur** [comment]
As mentioned above, in our practice we cannot reproduce the issue, as NVDA cannot read content of IntelliJ IDEA at all.

- Importance of experience: it depends, varies from somewhat important to important.
- Quality of experience: not impacted. Since the Narrator is not experiencing this kind of issues, users can use the Narrator to as an alternative workaround.
- Alternate solutions: No alternate technologies or options can be leveraged.
- Evidence: All evidence is provided in this issue and external issue.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749977
